src/formatting_stack/reporters/pretty_printer.clj

```diff
@@ -37,13 +38,14 @@
                               :exception " exceptions occurred"
                               :error     " errors found"
                               :warning   " warnings found"))
-                     (colorize (case type
-                                 :exception :red
-                                 :error     :red
-                                 :warning   :yellow))
+                     (cond-> colorize?
```

Comment:
`(-> ... (cond->))` is pretty confusing, as the inner `cond->` must be read differently to its more usual shape

a single enclosing `(cond->)` with some `true` clauses would preserve the shape people are more used to